File: Data Structures & Algorithms/binary-search/submission-0.cpp
class Solution {
public:
        int search(vector<int>& v, int t) {
        int l=0,r=v.size()-1,m;
        while(l<=r)
        {
            m=l+(r-l)/2;
            if(v[m]==t)
                return m;
            else if(v[m]<t)
                l=m+1;
            else
                r=m-1;
        }
        return -1;
    }
};

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.java
class Solution {
    public int longestConsecutive(int[] nums) {
                int l=0,m=0;
                Set<Integer> set = new HashSet<>();
                for (int i = 0; i < nums.length; i++) {
                        set.add(nums[i]);
                }
                for (int i = 0; i < nums.length; i++) {
                        if(!set.contains(nums[i] - 1)) {
                               l=1;
                               int j= nums[i] + 1;
                               while (set.contains(j)) {
                                       j++;
                                       l++;
                               }
                               m=Integer.max(l,m);
                        }
                }
                return m;
        }
}
